presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoStatement.java

```diff
@@ -36,6 +38,9 @@
 public class PrestoStatement
         implements Statement
 {
+    // Keep in sync with SystemSessionProperties.QUERY_MAX_RUN_TIME
```

Comment:
No need for this comment or constant. We can inline this since it is public (we can't change it without breaking users, similar to how we can't rename the `SELECT` keyword or a function)